src/main/scala/uk/gov/nationalarchives/dp/client/ContentClient.scala

```diff
@@ -1,22 +1,20 @@
 package uk.gov.nationalarchives.dp.client
 
 import cats.MonadError
-import cats.implicits._
 import cats.effect.Sync
+import cats.implicits._
 import sttp.client3._
 import sttp.client3.upicklejson.asJson
-import sttp.model.Method
-import uk.gov.nationalarchives.dp.client.DataProcessor.ClosureResultIndexNames
+import sttp.model.Uri
 import uk.gov.nationalarchives.dp.client.Client.ClientConfig
+import uk.gov.nationalarchives.dp.client.ContentClient.SearchQuery
 import uk.gov.nationalarchives.dp.client.Entities._
 import upickle.default._
 
-import java.time.ZonedDateTime
-import java.time.format.DateTimeFormatter
 import java.util.UUID
 
 trait ContentClient[F[_]] {
-  def findExpiredClosedDocuments(): F[List[Entity]]
+  def search(searchQuery: SearchQuery, max: Int = 100): F[List[Entity]]
```

Comment:
I think `searchEntities` is ok. It shouldn't just be looking for the title, I've changed that and I'd rather keep it as search rather than find because it matches the API call.